src/modules/tasks.js

```diff
@@ -0,0 +1,44 @@
+const tasks = [
+  {
+    description: 'Read an article',
+    completed: false,
+    id: 0,
+  },
+  {
+    description: 'Watch Andrew Wommack',
+    completed: false,
+    id: 1,
+  },
```

Comment:
- [ ] On page load, the tasks should appear in order of the `index` values for each task. I strongly suggest you set the property index to the value of the new array length (i.e. if you have 5th task to the list, the index of that task should equal 5). HINT thought_balloon you can modify indexes of your code and make sure the first index is 1 and not zero  this way when you create a todo list, the index starts from 1 instead of 0